cmd/tink-worker/internal/worker.go

```diff
@@ -1,27 +1,30 @@
-package main
+package internal // import "github.com/tinkerbell/tink/cmd/tink-worker/internal"
```

Comment:
this is the first import statement in this repo, we should discuss this separately and/or have it in every file right?

Reply:
fair enough, I'll remove the `import` comment. I had some early problems getting this internal package to work (ultimately unrelated) and I became very explicit in the process.

Reply:
https://golang.org/pkg/cmd/go/#hdr-Import_path_checking